parm/config/config.wave

```diff
@@ -24,6 +29,7 @@ export CDUMPRSTwave="gdas"
 export waveGRD='gnh_10m aoc_9km gsh_15m'
 export waveGRDN='1 2 3' # gridnumber for ww3_multi
 export waveGRDG='10 20 30' # gridgroup for ww3_multi
+export USE_WAV_RMP='NO' #yes/no rmp grid remapping pre-processed coefficients
```

Comment:
Since all other variables for waves are still for the GFSv16/develop case, this variable should be YES

Reply:
Am changing this to be set to YES, thanks!